### PR TITLE
Homepageui

### PR DESCRIPTION
**Pull request by ramblelob**
Asana Ticket: Create Landing Page with Login Button [https://app.asana.com/0/home/1201082019605980/1202544183788113](url)

Created landing page featuring the Tinder logo,  "Swipe Right®" centered, Log In button, and Create Account button (changing gradient hover effect).

Screen Shot 2022-07-12 at 9 12 45 PM

Screen Shot 2022-07-12 at 9 14 39 PM

### CODE REVIEW

**Review by milkdudss — APPROVED** [review]
Looks good to me, nice work. Next time can you add couple people as reviewers? The option is usually on the right side of the page after you raise a PR. please and thank you. Also the link to the ticket wasn't working when I clicked on it, maybe that is just me though.

File: src/App.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import Dashboard from './pages/Dashboard';
 import Onboarding from './pages/Onboarding';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
 
+
```
Comment (Zyanvel) [inline src/App.js]: Nit: Could you please delete this extra space?

File: src/components/Nav.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import whiteLogo from '../images/white-logo-tinder.png';
+import colorLogo from '../images/color-logo-tinder.png';
+
```
Comment (Zyanvel) [inline src/components/Nav.js:4]: Nit: Could you please remove this extra space?

File: src/components/Nav.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+function Nav({minimal, authToken}) {
+
+    // const minimal = true;
```
Comment (Zyanvel) [inline src/components/Nav.js]: Nit: Could you please remove this variable since you are not currently using it?

File: src/components/Nav.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+  return (
+    <nav>
+        <div className='logo-container'>
+            <img className='logo' src={minimal ? colorLogo : whiteLogo} />
+
```
Comment (Zyanvel) [inline src/components/Nav.js:11]: Nit: Could you please remove this extra space?

File: src/components/Nav.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+        </div>
+        {!authToken && !minimal && <button className='nav-button'>Log in</button>}
+
```
Comment (Zyanvel) [inline src/components/Nav.js:17]: Nit: Could you please remove this extra space?

Comment (Zyanvel) [inline src/components/Nav.js:17]: Nit: Could you please remove this extra space?

File: src/pages/Home.js [hunk 6]
```diff
@@ -1,9 +1,27 @@
...
   return (
-    <div>Home</div>
+    <div className='overlay'>
+    <Nav minimal={false} authToken={authToken}/>
+      <div className="home">
+        <h1>Swipe Right®</h1>
+        <button className='primary-button' onClick= {handleClick}>
+          {authToken ? 'Signout' : 'Create Account'}
+        </button>
+
```
Comment (Zyanvel) [inline src/pages/Home.js]: Nit: Could you please remove this extra space?

File: src/pages/Home.js [hunk 7]
```diff
@@ -1,9 +1,27 @@
...
+      </div>
+
```
Comment (Zyanvel) [inline src/pages/Home.js]: Nit: Could you please remove this extra space?

File: src/pages/Home.js [hunk 8]
```diff
@@ -1,9 +1,27 @@
-import React from 'react'
+import React from 'react';
+import Nav from '../components/Nav';
+
```
Comment (Zyanvel) [inline src/pages/Home.js:3]: Nit: Could you please remove this extra space?

File: src/components/Nav.js (hunk 5 above)
Comment (Zyanvel) [inline src/components/Nav.js]: Nit: Could you please remove this extra space?

**Review by Zyanvel — APPROVED** [review]
Great work and I love you were able to resolve the color gradient! I left some nit comments that would be nice to fix before you merge the pull request, but I have already approved it so you don't need to wait for that. 😄  PS: I also couldn't see the Asana ticket.

**Comment by ramblelob** [comment]
Hello, I pushed the revised branch with the corrections.

**Pull request merged**